public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/schedule-grid.js

```diff
@@ -94,17 +94,17 @@ function groupSessionsByDate( sessions ) {
  * Render the schedule for a specific day.
  *
  * @param {Object} props
- * @param {string} props.date     In a format acceptable by `wp.date.dateI18n()`.
- * @param {Array}  props.sessions
+ * @param {string} props.localDate The date being displayed (in the site's timezone).
```

Comment:
```suggestion
 * @param {string} props.localDate The day being displayed in Ymd format (in the site's timezone).
```

Just to clarify that this isn't a timestamp.

Reply:
👍🏻